### ISSUE TITLE
Custom html in our xml comments on controllers are not being displayed as html.

### ISSUE THREAD

**Issue by ThinkAboutGitHub** [domaindrivendev/Swashbuckle.AspNetCore#371]
We have a problem with our generated documentation, in that previously we had html images defined in the xml comments on controller methods.  This html was usually something like an image.
These images are now being displayed as text html, rather than displaying the icons that were working previously.
Is there something different in the way that this works now or is this a bug?

**Comment by ThinkAboutGitHub** [comment]
Has anyone had a chance to look into this?

**Comment by Britford** [comment]
It would be helpful if Swashbuckle rendered HTML

**Comment by domaindrivendev** [comment]
This appears to be resolved with the latest version (v2.5.0 as of writing).

```csharp
// ProductsController.cs
/// <summary>
/// Create a new Product
/// </summary>
/// <remarks>
/// Image: <img src="/foobar.png" />
/// </remarks>
[HttpPost]
[ProducesResponseType(201)]
[ProducesResponseType(typeof(ErrorResponse), 400)]
public IActionResult Create([FromBody, Required]Product product)
```

![screenshot](https://user-images.githubusercontent.com/1434343/41322476-f8ad835c-6e5d-11e8-8c5c-b7c03725a292.png)
